yelp_package/dockerfiles/hacheck-sidecar/Dockerfile

```diff
@@ -3,9 +3,10 @@ FROM docker-dev.yelpcorp.com/xenial_yelp
 ARG PIP_INDEX_URL=https://pypi.yelpcorp.com/simple
 ENV PIP_INDEX_URL=$PIP_INDEX_URL
 
-RUN apt-get update && apt-get install -y hacheck python paasta-tools=0.81.14-yelp1
+RUN apt-get update && apt-get install -y hacheck python paasta-tools
```

Comment:
I'm not so sure about unpinning paasta-tools, though it depends on the deployment process for the sidecar. I just don't want any surprises if paasta-tools has to be rolled back and we forget the sidecar needs to be rebuilt. Since we only have the docker image is the only artifact, it's not clear what is installed in the sidecar at a glance.

This actually leads me to realize hacheck and python have the same problem.

Reply:
Yeah I thought about this, but I don't know yet which paasta_tools version will contain this change. I could re-pin afterwards though.
Note that the hacheck sidecar container image needs to be [manually built and pushed](https://github.com/Yelp/paasta/blob/master/yelp_package/dockerfiles/hacheck-sidecar/Makefile#L6).